specification/digitaltwins/resource-manager/Microsoft.DigitalTwins/stable/2020-12-01/examples/DigitalTwinsGet_WithPrivateEndpointConnection_example.json

```diff
@@ -0,0 +1,43 @@
+{
+    "parameters": {
+      "subscriptionId": "50016170-c839-41ba-a724-51e9df440b9e",
+      "resourceGroupName": "resRg",
+      "resourceName": "myDigitalTwinsService",
+      "api-version": "2020-12-01"
+    },
+    "responses": {
+      "200": {
+        "body": {
+          "id": "/subscriptions/50016170-c839-41ba-a724-51e9df440b9e/resourcegroups/resRg/providers/Microsoft.DigitalTwins/digitalTwinsInstances/myDigitalTwinsService",
+          "location": "westus2",
+          "type": "Microsoft.DigitalTwins/digitalTwinsInstances",
+          "name": "myDigitalTwinsService",
+          "properties": {
+            "createdTime": "2019-11-19T12:55:05.229Z",
```

Comment:
lets align to the systemMetadata - https://armwiki.azurewebsites.net/rpaas/systemdata.html?q=system

Reply:
may be in a later api-version

Reply:
We will do this in the next API version.